README.md

```diff
@@ -3,30 +3,41 @@
 [![Build Status][build-badge]][build-link]
 [![Apache 2.0 License][license-badge]][license-link]
 
-The SDK has dependencies on specific versions of the C++ WebAssembly toolchain Emscripten (https://emscripten.org) and the protobuf library, therefor use of a Docker image is recommended.
+The SDK uses [bazel](https://bazel.build/) for its build. 
 
-## Docker
 
-A Dockerfile for the C++ SDK is provided in Dockerfile-sdk.
+## Creating a project for use
 
-It can built in this directory by:
+Create a directory with your WORKSPACE file, source files, and BUILD files:
```

Comment:
Please add `` around `WORKSPACE` and `BUILD`.

Reply:
Done.